src/components/Container/Container.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Container from '../Container/Container';
+import ContactForm from '../ContactForm/ContactForm';
+import Filter from '../Filter/Filter';
+import ContactList from '../ContactList/ContactList';
+
+describe('Filter component', () => {
+  const props = {
+    children: [
+      <h1 key="title-1">Phonebook</h1>,
+      <ContactForm key="contact-form" />,
+      <h2 key="title-2">Contacts</h2>,
+      <Filter key="filter" />,
+      <ContactList key="contact-list" />,
+    ],
+  };
+
+  const component = shallow(<Container {...props}></Container>);
```

Comment:
check if your component has all children's that you expect

Reply:
Test with checking of the each children rendering separately was added.